yamux/src/session.rs

```diff
@@ -138,6 +138,7 @@ where
 {
     /// Create a new session from a low level stream
     pub fn new(raw_stream: T, config: Config, ty: SessionType) -> Session<T> {
+        assert!(config.max_stream_window_size >= crate::config::INITIAL_STREAM_WINDOW);
```

Comment:
Please mention this limitation in function doc.

Reply:
add in config doc